app/components/dashboard/on_prem_status_component.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="card border-0 text-start">
+  <div class="card-body" />
+    <div class="card-title fs-4 fw-semibold d-flex justify-content-between">
+      Moabs on Prem
```

Comment:
Can we use labeling that is consistent with the code, with additional comments for context, e.g., ("Moabs on storage (i.e., on prem)")?

Reply:
Aside: I feel like part of what is confusing about PresCat is that we are inconsistent about terminology across codebase, docs, etc.

Reply:
agree - and I stupidly forgot that you had already picked a term to use!